src/gauss3.jl

```diff
@@ -0,0 +1,96 @@
+#=
+gauss3.jl
+=#
+
+
+#using ImagePhantoms #: Object, Object3d
+
+export Gauss3
+export phantom, radon, spectrum
+
+
+"""
+    Gauss3 <: AbstractShape3
+"""
+struct Gauss3 <: AbstractShape3 end
+
+
+# constructors
+
+
+"""
+    Gauss3(cx, cy, cz, wx, wy, wz, Φ=0, Θ=0, value::Number = 1)
+    Gauss3(center::NTuple{3,RealU}, radii::NTuple{3,RealU}=(1,1,1), angle::NTuple{2,RealU}=(0,0), v=1)
+    Gauss3([9-vector])
+    Gauss3(r, v=1) (isotropic of width `w`)
+Construct `Gauss3` object from parameters;
+here `width` = FWHM (full-width at half-maximum).
+"""
+function Gauss3(
+    cx::RealU,
+    cy::RealU,
+    cz::RealU,
+    wx::RealU,
+    wy::RealU = wx,
+    wz::RealU = wx,
+    Φ::RealU = 0,
+    Θ::RealU = 0,
+    value::Number = 1,
+)
+    (cx, cy, cz, wx, wy, wz) = promote(cx, cy, cz, wx, wy, wz)
+    Object(Gauss3(), (cx,cy,cz), (wx,wy,wz), (Φ, Θ), value)
```

Comment:
I really appreciate this feedback because I find it confusing too!
This package is still in 0.0.x stage so I am open to modifying it to make it more "usual."
I was attempting to follow what base does with `round` but probably I didn't match it properly:
https://github.com/JuliaLang/julia/blob/742b9abb4dd4621b667ec5bb3434b8b3602f96fd/base/rounding.jl#L52

Would you recommend simply changing to lower case `gauss3` and `ellipsoid` etc., or is there a better overall way?
BTW, I still plan to merge this particular PR, but I can consider your suggestion in a separate PR that could change it across the board.